### PR TITLE
Get logits

### PR DESCRIPTION
**Pull request by xipingyan**
Internal test, just for get logits.

### CODE REVIEW

**Comment by Wovchena** [comment]
Feel free to reopen if needed
